Client/Frontend/Settings/NewTabContentSettingsViewController.swift

```diff
@@ -27,7 +27,13 @@ class CurrentTabSetting: Setting {
 
 class NewTabContentSettingsViewController: SettingsTableViewController {
 
-    init() {
+    /* variables for checkmark settings */
+    let prefs: Prefs
+    var currentChoice: NewTabPage!
+    var hasHomePage: Bool!
```

Comment:
Try not to use implicit unwrapped optional (or force unwraps in other places). We do this on member vars in some UIViewController code because Apple made it a de-facto standard to do implicit unwrapped optionals for the objects instantiated in `viewDidLoad()`. Even in that case we are trying to get out of that habit, because as the class complexity grows, the risk that code calls an implicitly unwrapped optional prior to initialization increases.